apps/contributor/contracts/Contributor.sol

```diff
@@ -60,23 +60,23 @@ contract Contributor is AragonApp {
     ContributorAccountUpdated(id, oldAccount, newAccount);
   }
 
-  function updateContributorIpfsHash(uint32 id, bytes32 ipfsHash, uint8 hashFunction, uint8 hashSize) public isInitialized auth(MANAGE_CONTRIBUTORS_ROLE) {
+  function updateContributorIpfsHash(uint32 id, bytes32 hashDigest, uint8 hashFunction, uint8 hashSize) public isInitialized auth(MANAGE_CONTRIBUTORS_ROLE) {
     Contributor storage c = contributors[id];
-    bytes32 oldIpfsHash = c.ipfsHash;
-    c.ipfsHash = ipfsHash;
+    bytes32 oldHashDigest = c.hashDigest;
+    c.hashDigest = hashDigest;
     c.hashFunction = hashFunction;
     c.hashSize = hashSize;
 
-    ContributorProfileUpdated(id, oldIpfsHash, c.ipfsHash);
+    ContributorProfileUpdated(id, oldHashDigest, c.hashDigest);
```

Comment:
General question: When do you need to use `emit` for an event?

Reply:
always in newer solidity version. events without emit is deprecated - so we must update that soon.

Reply:
Thanks!